tutorials/Tutorial_NumericalModel_3D.jl

```diff
@@ -0,0 +1,95 @@
+#=
+# Creating 3D numerical model setups
+
+### Aim
+The aim of this tutorial is to show you how to create 3D numerical model setups that can be used as initial setups for other codes.
+
+=#
+
+
+#=
+### 3D Subduction setup
+
+Lets start with creating a 3D model setup in cartesian coordinates, which uses the `CartData` data structure
+=#
+using GeophysicalModelGenerator
+
+nx,ny,nz = 512,512,128
+x = range(-1000,1000, nx);
+y = range(-1000,1000, ny);
+z = range(-660,0,    nz);
+Grid = CartData(XYZGrid(x,y,z));
+
+# Now we create an integer array that will hold the `Phases` information (which usually refers to the material or rock type in the simulation)
+Phases = fill(2,nx,ny,nz);
+
+# In many (geodynamic) models, one also has to define the temperature, so lets define it as well
+Temp = fill(1350.0, nx,ny,nz);
+
+# #### Simple free subduction setup
+
+# Much of the options are explained in the 2D tutorial, which can directly be transferred to 3D.
+# Therefore, we will start with a simple subduction setup, which consists of a horizontal part that has a mid-oceanic ridge on one explained
+
+# We use a lithospheric structure. Note that if the lowermost layer has the same phase as the mantle, you can define `Tlab` as the lithosphere-asthenosphere boundary which will automatically adjust the phase depending on temperature
+lith = LithosphericPhases(Layers=[15 45 10], Phases=[0 1 2], Tlab=1250)
```

Comment:
Perhaps I'm missing where this is explained, but it was not clear to me what `Layers=[15 45 10]`  means. It's the depth distance from the previous layer right?

Reply:
I have extended that